docs/plugins/dotnet-plugin-events-reference.md

```diff
@@ -0,0 +1,1833 @@
+---
+id: dotnet-plugin-events-reference
+title: .NET Plugin Events - Reference
+sidebar_label: .NET Plugin Events - Reference
+slug: /plugins/dotnet-plugin-events-reference
+toc_max_heading_level: 2
+---
+
+# .NET Plugin Events - Reference
```

Comment:
I tend to think that this page (and the Node.js folder and files) should be under the "Custom Plugin Arch" folder.
WDYT?